packages/client/src/components/devtools/DevToolsHeader.svelte

```diff
@@ -31,12 +32,12 @@
   onMount(async () => {
     // make sure correct before starting
     await devToolsStore.actions.changeRole(SELF_ROLE)
-    staticRoleList = await roleStore.actions.fetchAccessibleRoles()
+    roles = await API.getRoles()
   })
 </script>
 
 <div class="dev-preview-header" class:mobile={$context.device.mobile}>
-  <Heading size="XS">Preview</Heading>
+  <Heading size="XS">Previ werewr eww</Heading>
```

Comment:
🥚

Reply:
Thank you 😂  May have been testing things...